orquesta/conducting.py

```diff
@@ -882,3 +885,40 @@ def get_task_transition_contexts(self, task_id, route):
                 contexts[task_transition_id] = self.get_task_initial_context(t[1], route)
 
         return contexts
+
+    def request_rerun_workflow_status(self, status, rerun_options):
```

Comment:
Please refactor this to `def request_workflow_rerun(self, options):`. The conductor decides what status to set for the workflow.